__tests__/lib/dev-environment/dev-environment-core.js

```diff
@@ -25,22 +25,28 @@ import { getEnvironmentPath,
 import { searchAndReplace } from '../../../src/lib/search-and-replace';
 import { resolvePath } from '../../../src/lib/dev-environment/dev-environment-cli';
 import { DEV_ENVIRONMENT_NOT_FOUND } from '../../../src/lib/constants/dev-environment';
+import { bootstrapLando } from '../../../src/lib/dev-environment/dev-environment-lando';
 
 jest.mock( 'xdg-basedir', () => ( {} ) );
-jest.mock( 'fs' );
 jest.mock( '../../../src/lib/api/app' );
 jest.mock( '../../../src/lib/search-and-replace' );
 jest.mock( '../../../src/lib/dev-environment/dev-environment-cli' );
 
 describe( 'lib/dev-environment/dev-environment-core', () => {
+	const cleanup = () => fs.rmSync( path.join( os.tmpdir(), 'lando' ), { recursive: true, force: true } );
+
+	beforeAll( cleanup );
+	afterAll( cleanup );
+
 	beforeEach( () => {
-		jest.clearAllMocks();
+		jest.resetAllMocks();
+		jest.restoreAllMocks();
```

Comment:
Ensure that we clear what we have mocked after every test. This way, our tests won't depend on which functions the previously run tests have mocked. All tests must be isolated. While this leads to slightly more code, we get reproducible and predictable behavior in return.